api/client/lib.py

```diff
@@ -220,6 +220,17 @@ def get_data(url, headers, params=None, logger=None):
         url, retry_count, data))
 
 
+@memoize(maxsize=None)
+def allowed_units(access_token, api_host, metric_id, item_id):
```

Comment:
For the sake of consistency with other functions, I think this should be named `get_allowed_units`. See:

* `get_available`
* `get_data_series`
* `get_data_points`
* `get_geo_centre`
* `get_geojson`
* `get_descendant_regions`

Reply:
done